pennylane/devices/tests/test_measurements.py

```diff
@@ -1628,7 +1648,7 @@ def test_sample_measurement_without_shots(self, device):
         """Test that executing a sampled measurement with ``shots=None`` raises an error."""
         dev = device(2)
 
-        if dev.shots is not None:
+        if bool(dev.shots):
```

Comment:
Why use `bool(dev.shots)` instead of just `dev.shots` like before?

Reply:
```suggestion
        if dev.shots:
```